src/tuner/nccl_ofi_tuner.c

```diff
@@ -295,8 +295,43 @@ ncclResult_t nccl_ofi_tuner_init(size_t nRanks, size_t nNodes, ncclDebugLogger_t
 	return ret;
 }
 
-ncclResult_t nccl_ofi_tuner_get_coll_info(
-	void *context, ncclFunc_t collType, size_t nBytes, int collNetSupport, int nvlsSupport, int numPipeOps, int *algorithm, int *protocol, int *nChannels)
+void nccl_ofi_tuner_disable(float **collCostTable, int algorithm, int protocol, int numAlgo, int numProto)
+{
+	float(*table)[NCCL_NUM_PROTOCOLS] = (float(*)[NCCL_NUM_PROTOCOLS])collCostTable;
+
+	float long_time = 3600000000.0;  // 1 hour
```

Comment:
Challenge accepted :)

Jokes aside, were you able to get confirmation from code on a plugin setting -1/an enum val back instead of doing this? Do we need any more clarification from the NCCL team?

Reply:
`NCCL_ALGO_PROTO_IGNORE` is -1, I asked and they weren't sure if it was ok to set it in the plugin or not, but for sure it is ok to set it to a large value.

Reply:
Ideally, this would be `FLT_MAX`.

Reply:
I mainly chose this value to follow what NCCL does. 
https://github.com/NVIDIA/nccl/blob/178b6b759074597777ce13438efb0e0ba625e429/src/enqueue.cc#L1586
Values higher than this would not be selected by NCCL anyway given how they select the minimum.

Reply:
> NCCL_ALGO_PROTO_IGNORE is -1, I asked and they weren't sure if it was ok to set it in the plugin or not, but for sure it is ok to set it to a large value.

Ack.